software/28.Gin/database/database.json

```diff
@@ -0,0 +1,32 @@
+[
+ {
+  "name": "objet2",
+  "description": "un autre objet",
+  "price": 20
+ },
+ {
+  "name": "objet3",
+  "description": "un objet de test",
+  "price": 30
+ },
+ {
+  "name": "objet9",
+  "description": "le objet",
+  "price": 10
+ },
+ {
+  "name": "objetBETA",
+  "description": "un bel objet",
+  "price": 10
+ },
+ {
+  "name": "objetALPHA",
+  "description": "un bel objet",
+  "price": 10
+ },
+ {
+  "name": "objet10",
+  "description": "un bel objeEEEt",
+  "price": 10
+ }
+]
```

Comment:
Use English words it's better

Reply:
Yes it has been modified in the correction just forgot to change it in the subject